src/components/SetupPanes/FindStrictReceivePaymentPaths.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+
+import OptionsTablePair from "../OptionsTable/Pair";
+import PubKeyPicker from "../FormComponents/PubKeyPicker.js";
+import AssetPicker from "../FormComponents/AssetPicker.js";
+import AmountPicker from "../FormComponents/AmountPicker.js";
+import AssetMultiPicker from "../FormComponents/AssetMultiPicker.js";
+
+export default function FindStrictReceivePaymentPaths({ values, onUpdate }) {
+  return (
+    <div>
+      <OptionsTablePair label="Source Account">
+        <PubKeyPicker
+          value={values["source_account"]}
+          onUpdate={(value) => {
+            onUpdate("source_account", value);
+          }}
+        />
+      </OptionsTablePair>
+      <OptionsTablePair label="Source Assets">
+        <AssetMultiPicker
+          stringForm
+          value={
+            values["source_assets"] === undefined
+              ? []
+              : // Filter out empty strings from resulting array.
+                // ''.split(',') produces ['']
+                values["source_assets"].split(",").filter(Boolean)
```

Comment:
I kinda don't like how this is, but I don't particularly want redux to have to care about munging the data it receives, and I also don't want `AssetMultiPicker` to emit a single comma-separated list. Least-bad option, though the `.filter(Boolean)` makes it a lot more bad.

Reply:
ah yeah,`.filter(Boolean)` is kind of a neat trick - wondering if it's worth being a bit more verbose rather than explaining w comment

values["source_assets"].split(",").filter(str => str.length)

but yeah, nbd

Reply:
Yeah I think I'd still want to have a comment explaining what produces an empty string in the array, so 🤷‍♂